mapbox_vector_tile/polygon.py

```diff
@@ -196,7 +200,7 @@ def make_valid_multipolygon(shape):
     return MultiPolygon(new_g)
 
 
-def make_it_valid(shape):
+def make_it_valid(shape, asserted=True):
```

Comment:
This change does not change the behaviour of `make_it_valid(shape)`, the default value `True` preserves the behaviour.